### PR TITLE
feat(apps): Apps create command with feature flag

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

meroxa apps create NAME

Fixes https://github.com/meroxa/turbine-project/issues/42

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [x]  Tested in minikube

## Demo

Screenshot from 2022-02-25 14-19-02

### CODE REVIEW

File: cmd/meroxa/root/apps/create.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+type Create struct {
+	client createApplicationClient
+	logger log.Logger
+
+	args struct {
+		Name string
+	}
+
+	flags struct {
```
Comment (raulb) [inline cmd/meroxa/root/apps/create.go:46]: We don't need a path since we shouldn't be doing anything with it.

Comment (raulb) [reply cmd/meroxa/root/apps/create.go:46]: Looks like you mean `language` ?

**Review by raulb — CHANGES_REQUESTED** [review]
Need to change the flag name (it shouldn't be path, but language instead)

**Review by raulb — APPROVED** [review]

File: cmd/meroxa/root/apps/create.go [hunk 2]
```diff
@@ -61,15 +65,31 @@ func (c *Create) Flags() []builder.Flag {
...
-		c.args.Name = args[0]
+	if len(args) < 1 {
+		return errors.New("requires an application name")
 	}
+	c.args.Name = args[0]
 	return nil
 }
 
+func (c *Create) getLang() (string, error) {
+	if path := c.flags.Path; path != "" {
```
Comment (janelletavares) [inline cmd/meroxa/root/apps/create.go:76]: what if path is pwd... we still need that to be specified as a flag?

Comment (raulb) [reply cmd/meroxa/root/apps/create.go:76]: > what if path is pwd... we still need that to be specified as a flag?

@janelletavares That's a great question. I'm making specifying the path mandatory (notice I'm not reading from pwd) but maybe we could make that change.

> what if the supplied language is different from the app.json language?

Language specified in the app.json currently takes precedence (i.e. if both provided we ignore the currently ignore `--lang`

**Pull request merged**